Parsers/Binary To Decimal Converter.js

```diff
@@ -0,0 +1,47 @@
+/*
+activation_example:!converttoDecimal <binary_code>
+regex:^!converttoDecimal 
+flags:i
+*/
+
+// get the user input in format !converttoDecimal binary_code for example - !converttoDecimal 1011
+var input = current.text.trim(); 
+
+
+// Extract the Binary expression from the input
+var binaryCheckregx = /^[01]+$/; // regular expresion for valid binary code
+var match = input.match(/!converttoDecimal (\d+)/); // check input format 
+var expression = match ? match[1] : "";
```

Comment:
It's not clear from other details about this parser, that this will not support multiple bytes (i.e. `00110101 11111111`) or breaks in the binary

Reply:
Ok I will rectify the code